spec/rubocop/cop/lint/redundant_type_conversion_spec.rb

```diff
@@ -285,6 +285,10 @@
     it_behaves_like 'offense', :to_h, '::Kernel::Hash({ foo: bar })'
     it_behaves_like 'offense', :to_h, 'Hash[foo: bar]'
     it_behaves_like 'offense', :to_h, '::Hash[foo: bar]'
+
+    it_behaves_like 'accepted', 'Hash.new(hash).to_h { |k, v| [foo(k), bar(v)] }'
```

Comment:
Passing a hash to `Hash.new` doesn't seem like a very realistic example. I'd probably just use a literal instead.

Reply:
I've updated this example to use hash literal.